src/deployers/MultimediaPluginsDeployer.cpp

```diff
@@ -17,18 +17,31 @@ bool MultimediaPluginsDeployer::deploy() {
     if (!BasicPluginsDeployer::deploy())
         return false;
 
-    ldLog() << "Deploying mediaservice plugins" << std::endl;
+    if (fs::exists(qtPluginsPath / "mediaservice")) {
```

Comment:
Do we need this check at all? Is there a chance there are no such plugins to be deployed? I'd consider that an error, actually. (Maybe we just need a warning log message in some `else` block.)

Reply:
In Qt 5, mediaservice and audio exist, while in Qt 6 multimedia may exist (depending on the Qt 6 version, as I think earlier versions of Qt 6 did not have this).

I can add warning messages when these are not found.